thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -159,109 +159,114 @@
       {{/if}}
 
       {{!-- Alert anomaly table --}}
-      <table class="te-anomaly-table">
-        {{#if filteredAnomalies}}
-          <thead>
-            <tr class="te-anomaly-table__row te-anomaly-table__head">
-               <th class="te-anomaly-table__cell-head">
-                <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "start"}}>
-                  Start/Duration (PDT)
-                  <i class="te-anomaly-table__icon glyphicon {{if sortColumnStartUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
-                </a>
-               </th>
-               {{#if alertHasDimensions}}
-                  <th class="te-anomaly-table__cell-head te-anomaly-table__cell-head--fixed">Dimensions</th>
-               {{/if}}
-               <th class="te-anomaly-table__cell-head">
-                <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "score"}}>
-                  Severity Score
-                  <i class="te-anomaly-table__icon glyphicon {{if sortColumnScoreUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
-                </a>
-               </th>
-               <th class="te-anomaly-table__cell-head">
-                <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "change"}}>
-                  {{baselineTitle}}
-                  <i class="te-anomaly-table__icon glyphicon {{if sortColumnChangeUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
-                </a>
-               </th>
-               <th class="te-anomaly-table__cell-head">
-                <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "resolution"}}>
-                  Resolution
-                  <i class="te-anomaly-table__icon glyphicon {{if sortColumnResolutionUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
-                </a>
-             </th>
-            </tr>
-          </thead>
+      <div class="te-block-container">
```

Comment:
I'm seeing a lot of work done to the table. I wonder if going back to the `ember-models-table` would save us time in the long run? (not a blocker!)

Reply:
Yes. We likely will. But all I'm doing here is adding a container div in order to position the loader icon.